src/Riok.Mapperly/MapperGenerator.cs

```diff
@@ -83,6 +83,13 @@ CancellationToken cancellationToken
             if (!symbolAccessor.HasAttribute<MapperAttribute>(mapperSymbol))
                 continue;
 
+            if (symbolAccessor.HasMultipleAssemblyAttribute<DefaultMapperAttribute>())
+            {
+                diagnostics.Add(
+                    Diagnostic.Create(DiagnosticDescriptors.CannotUseMultipleAssemblyAttributes, null, typeof(DefaultMapperAttribute))
```

Comment:
Doesn't this emit the diagnostic for each mapper class instead of once per assembly?